paasta_tools/chronos_tools.py

```diff
@@ -257,6 +257,13 @@ def get_disabled(self):
     def get_schedule(self):
         return self.config_dict.get('schedule')
 
+    def get_schedule_interval_in_seconds(self):
+        try:
+            _, _, interval = self.get_schedule().split('/')
```

Comment:
sorry, I should've been clearer. This isn't anything to do with the crontab program, but we support specifying schedules for jobs in both ISO8601 notation, and the same format as cron. This means we need to support the functionality you're adding when your schedule is in both formats.